v1/api/swagger/components/schemas/OSServiceImage.yaml

```diff
@@ -0,0 +1,101 @@
+allOf:
+  - $ref: ./ResourceBase.yaml
+  - type: object
+    required:
+      - Description
+      - Category
+      - Flavor
+      - Version
+      - Origin
+      - Timeout
+      - DeviceLayouts
+      - PermittedProjects
+      - Files
+      - Info
+      - Approach
+      - AssumedBootMethod
+      - NoSwitchLAG
+      - BondMode
+      - FWBaselineID
+      - UserDefinedSteps
+      - Classifiers
+    properties:
+      Description:
+        type: string
+        example: Description of the image
+      Category:
+        type: string
+        example: linux
+        description: >-
+          A high level classification of the service. 
+      Flavor:
+        type: string
+        example: ubuntu
+        description: >-
+          An high level description of the OS image type. 
+      Version:
+        type: string
+        example: "20.04.5"
+        description: >-
+          A specific version of the service flavor.
+      Origin:
+        $ref: OSServiceImageOrigin.yaml
+      Timeout:
+        type: integer
+        format: int64
+        example: 1000
+        description: >- 
+          Maximum amount of time in seconds for the service to complete its work. 
+          This time includes the amount of time to download the service image if necessary.
```

Comment:
There's not a separate download timeout?

Reply:
Yes there is a separate download time. This is the total timeout which includes everything download, installation, etc..